v0/destinations/fb/transform.js

```diff
@@ -253,8 +248,11 @@ function buildBaseEvent(message) {
   } else if (isAppleFamily(sourceSDK)) {
     sourceSDK = "i2";
   } else {
-    // if the sourceSDK is not android or ios, send an empty string
-    sourceSDK = "";
+    // if the sourceSDK is not android or ios
+    throw new CustomError(
+      "Extended Device Information i.e device.type is required",
+      400
+    );
```

Comment:
This is a breaking change we should provide support for backward compatibility.

Reply:
The error from facebook is not quite understandable. Hence we are blocking at transformer with understandable error message. Either way this is a breaking change introduced by facebook